README.md

```diff
@@ -790,6 +790,24 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --npm-client-args [client]
+
+Pass extra arguments to npm client.
+
+```sh
+$ lerna bootstrap --npm-client=yarn --npm-client-args=--ignore-engine
```

Comment:
Unfortunately that's not how `execa` works, you need to pass an array of arguments (or push multiple, in this case) if there are multiple arguments.